src/sdk/create.ts

```diff
@@ -145,9 +145,20 @@ export function createBanner(opts: CreateWidgetOptions) {
 
   const language = getLanguageFromOptionsOrParent(opts);
 
+  const isDark =
+    opts.theme === "dark" ||
+    (opts.theme === "auto" && window.matchMedia && window.matchMedia("(prefers-color-scheme: dark)").matches);
+
+  let primaryColor = "#565656";
+  let secondaryColor = "#a2a2a2";
+  if (isDark) {
+    primaryColor = "#a2a2a2";
+    secondaryColor = "#565656";
+  }
+
   const els = el.style;
   els.position = "absolute";
-  els.bottom = "2px";
+  els.bottom = "4px";
```

Comment:
```suggestion
  els.bottom = "6px";
```
Nit: Feels a bit less cramped my opinion. Feel free to ignore though.
**4px**
<img width="329" height="85" alt="Image" src="https://github.com/user-attachments/assets/9e5d22a6-908f-417c-878a-4f79df339fa4" />
**6px**
<img width="331" height="83" alt="Image" src="https://github.com/user-attachments/assets/d15a3df5-5466-4cba-b758-6f8442adac88" />